src/lib/provable/int.ts

```diff
@@ -77,7 +77,7 @@ class UInt64 extends CircuitValue {
     return this.value.toString();
   }
   /**
-   * Turns the {@link UInt64} into a {@link BigInt}.
+   * Turns the {@link UInt64} into a BigInt.
```

Comment:
I'm curious why you removed the `BigInt` link?

Reply:
It would also be valid to turn this rule off like we did with `notExported` in #1993.  I think it is worth knowing if there is a typo in a link or if a public API is referencing private APIs, and resolving those, rather than ignoring.  But if it becomes too much effort, or too much value is lost by working around this rule, we can just turn it off too.